tests/test_config_handler.py

```diff
@@ -0,0 +1,22 @@
+from unittest.mock import patch
+
+import pytest
+
+from datasetinsights.io.config_handler import override_config
+from datasetinsights.io.exceptions import InvalidOverrideKey
+
+
+@patch("yacs.config.CfgNode")
+def test_override_config(mock_cfg):
+
+    override = "a.b.c=test1 c.a_b=test2"
+    override_token = ["a.b.c", "test1", "c.a_b", "test2"]
+    override_config(override=override, config=mock_cfg)
+    mock_cfg.merge_from_list.assert_called_with(override_token)
+
+
+@patch("yacs.config.CfgNode")
```

Comment:
Why do you need mock config here? Why not just create real instance of CfgNode and pass to override_config here?